src/sidecar/middlewares.ts

```diff
@@ -106,6 +106,15 @@ export class ErrorResponseMiddleware implements Middleware {
         return;
       }
 
+      // Special case: suppress error logs for schema registry RBAC-related 404s
+      if (
+        context.response.status === 404 &&
+        /gateway\/v1\/schema-registry\/[^/]+\/subjects\/[^/]+\/versions\/latest/.test(context.url)
```

Comment:
I think this regex needs some adjusting since we get the response from the sidecar internal SR redirect (?) that just goes to `:26636/subjects/<subject>`. Still seeing this error logging when opening message viewer from a CCloud topic:
<img width="1041" alt="image" src="https://github.com/user-attachments/assets/71ec79ea-3be8-4299-8aa6-ff35c1f065b6" />

Reply:
Ok, I tried updating the regex and generating an error a couple times. Second time seems to have worked: 

<img width="1252" alt="Screenshot 2025-07-09 at 2 47 01 PM" src="https://github.com/user-attachments/assets/db2bce54-3aa5-4e90-b075-089884818f7c" />